pandas/plotting/_core.py

```diff
@@ -1665,7 +1679,21 @@ def scatter(self, x, y, s=None, c=None, **kwargs):
             ...                       c='species',
             ...                       colormap='viridis')
         """
-        return self(kind="scatter", x=x, y=y, s=s, c=c, **kwargs)
+        s = kwargs.pop('s', None)
+        size = kwargs.pop('size', None)
+        if s is not None and size is not None:
+            raise TypeError("Specify exactly one of `s` and `size`")
```

Comment:
🤔  "exactly one" makes it sound like people need to pass exactly one - however, they're allowed to pass neither, right?

Reply:
This is correct, but I copyied this line from here [https://github.com/pandas-dev/pandas/blob/439906e07dbad5b7047d7e15e37feedd3ae03df8/pandas/tests/plotting/frame/test_frame_color.py#L228] and thought I keep the formulation.

Reply:
ok, that's fine then

Reply:
if you ping when you've added tests I'll take a closer look

Reply:
Ping. FYI: One Test is not successful, but should be in my opinion. It is the one change in `test_series.py`.

Reply:
can you try moving what you've added to a new test case to check if it still fails there?

Reply:
It does not fail if I split the test into two. I will update myy PR.

Reply:
@MarcoGorelli I want to ping one more time, because I think my tasks are done. As far as I see are my added tests successful. To be honest I don't understand the three failures in the pipeline and think that they are not related to my changes. What will happen with this PR in the next few days?

Reply:
yeah they're probably unrelated, don't worry about them

> What will happen with this PR in the next few days?

I'll take a closer look when I get a chance